tests/wpt/metadata/webaudio/idlharness.https.window.js.ini

```diff
@@ -1,6 +1,6 @@
 [idlharness.https.window.html]
   [BaseAudioContext interface: operation createChannelSplitter(unsigned long)]
-    expected: FAIL
+    expected: PASS
```

Comment:
For all tests that are passing right now, you can actually remove the entries instead of expecting them to PASS, because by default tests are expected to pass.